sendgrid_backend/mail.py

```diff
@@ -145,7 +145,9 @@ def _build_sg_mail(self, msg):
 
                 attachment.filename = filename
                 # todo: Read content if stream?
-                attachment.content = base64.b64encode(content)
+                if isinstance(content, str):
+                    content = content.encode()
+                attachment.content = base64.b64encode(content).decode()
```

Comment:
Why the additional `.decode()`?

Reply:
Because if the attachment content is not a str, the backend will fail when sending the email

Reply:
Cool thanks! Does this work against the sendgrid API in both python 2 and 3?

Reply:
Yes, I ran the tests in python 2 and 3.

Reply:
Thanks!!